client/mac_address.cpp

```diff
@@ -263,7 +263,6 @@ int get_mac_address(char* address) {
         }
         hw_addr = (struct ether_addr *)&(item->lifr_lifru.lifru_enaddr);  
 #endif
-        strcpy(address, ether_ntoa(hw_addr));
```

Comment:
This was a _temporary_ change, meant _only_ for you and _only_ to get to you a working client fast with minimal work and minimal loss of functionality while waiting for a proper way to fix this. The proper way to fix this is to find out how to get the MAC address on NetBSD and then implement it. Additionally, the #if - #elif should have an #else to catch the next OS that needs its own code.